packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -42,18 +42,20 @@ to {
 const StyledDropdownOverlay = styled(BaseBox)<{
   transition: FlattenSimpleInterpolation;
   onAnimationEnd: () => void;
+  isOpen: boolean;
 }>(
   (props) =>
     css`
       ${props.transition}
       transform: translateY(${makeSize(props.theme.spacing[3])});
       opacity: 0;
       z-index: 99;
+      pointer-events: ${props.isOpen ? 'all' : 'none'};
     `,
 );
 
 type DropdownOverlayProps = {
-  children: React.ReactNode;
+  children: React.ReactElement;
```

Comment:
is this same as ReactNode 🤔

Reply:
Nope, ReactNode is a superset that includes ReactElement. [More Info](https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement)